client.go

```diff
@@ -151,144 +152,159 @@ func NewBlockchainInfoClient(network string) Client {
 			Params: &chaincfg.TestNet3Params,
 		}
 	default:
-		panic(fmt.Sprintf("Unknown Network %s", network))
+		panic(NewErrUnsupportedNetwork(network))
 	}
 }
 
-func (client *client) GetUnspentOutputs(address string, limit, confitmations int64) Unspent {
+func (client *client) GetUnspentOutputs(context context.Context, address string, limit, confitmations int64) (Unspent, error) {
 	if limit == 0 {
 		limit = 250
 	}
-	for {
+	val, err := client.exponentialBackoff(context, func() (interface{}, error) {
```

Comment:
is there are reason why `exponentialBackoff` is defined on the `client` as a method? It could just be a stand-alone function.

Reply:
The `exponentialBackoff` function should _not_ return `(interface{}, error)`. It should only return `error`. The closure that is backing off can capture the value it wants to write to, and write to it. This prevents dodgy type conversions and is consistent with the use of backoff in other repos.

Reply:
done